test/upgrade/service_postupgrade_test.go

```diff
@@ -79,22 +79,6 @@ func TestBYORevisionPostUpgrade(t *testing.T) {
 	}
 }
 
-func configHasGeneration(clients *test.Clients, serviceName string, generation int) (bool, error) {
```

Comment:
I don't think these helpers should be removed, because the uncommented code will be re-enabled https://github.com/knative/serving/blob/94842702cd3f3f9c19aa5854cf0e4080e79153c0/test/upgrade/service_postupgrade_test.go#L37

Reply:
I knew there was something up with these, just couldn't find the relevant commit quickly enough. Backing these out.